ionic_langchain/tool.py

```diff
@@ -18,20 +35,32 @@ def __init__(self, sdk: Optional[IonicSDK] = None):
         else:
             self._sdk = IonicSDK()
 
-    def query(self, queries: str) -> list:
+    def query(
+        self,
+        query: Query,
```

Comment:
Does this work? The issue was that langchain has a really hard time with multi-input tools

Reply:
reverted back to using query_input 👍